pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```diff
@@ -1,125 +1,111 @@
-schema = 1
-
-[device]
-name = "lightning.kokkos"
-
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-        "LinearCombination",
-]
+schema = 2
 
 # The union of all gate types listed in this section must match what
 # the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "Identity",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "S",
-        "T",
-        "PhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "Rot",
-        "CNOT",
-        "CY",
-        "CZ",
-        "SWAP",
-        "CSWAP",
-        "Toffoli",
-        "IsingXX",
-        "IsingXY",
-        "IsingYY",
-        "IsingZZ",
-        "ControlledPhaseShift",
-        "CRX",
-        "CRY",
-        "CRZ",
-        "CRot",
-        "SingleExcitation",
-        "SingleExcitationPlus",
-        "SingleExcitationMinus",
-        "DoubleExcitation",
-        "DoubleExcitationPlus",
-        "DoubleExcitationMinus",
-        "MultiRZ",
-        "QubitUnitary",
-        "GlobalPhase",
-]
+[operators.gates.native]
+
+CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
+ControlledPhaseShift   = { properties = [ "invertible",                 "differentiable" ] }
+CPhase                 = { properties = [ "invertible",                 "differentiable" ] }
+ControlledQubitUnitary = { properties = [ "invertible",                                  ] }
+CRot                   = { properties = [ "invertible"                                   ] }
+CRX                    = { properties = [ "invertible",                 "differentiable" ] }
+CRY                    = { properties = [ "invertible",                 "differentiable" ] }
+CRZ                    = { properties = [ "invertible",                 "differentiable" ] }
+CSWAP                  = { properties = [ "invertible",                 "differentiable" ] }
+CY                     = { properties = [ "invertible",                 "differentiable" ] }
+CZ                     = { properties = [ "invertible",                 "differentiable" ] }
+DoubleExcitationMinus  = { properties = [ "invertible",                 "differentiable" ] }
+DoubleExcitationPlus   = { properties = [ "invertible",                 "differentiable" ] }
+DoubleExcitation       = { properties = [ "invertible",                 "differentiable" ] }
+# FIXME: device reports it is controllable. Why is that?
+GlobalPhase            = { properties = [ "invertible", "controllable", "differentiable" ] }
```

Comment:
Vincent commented this was not controllable, what was the resolution?

Reply:
Oh, right. Hmm, I think we need to hardcode a special case for schema 2 to pass validation in Catalyst then.

Reply:
Hmm, yes the toml is about the low-level capabilities. The validation is mainly there so the toml doesn't go out of sync too easily. M)aybe it would be better to go back to the simple validation we had before? (that is do not compare Adjoint(Gate) or C(Gate)

Reply:
I would suggest to introduce a smaller relaxation since it is the only exception we have. See https://github.com/PennyLaneAI/catalyst/pull/718

Reply:
Meanwhile I have removed the controllable flag here.